packages/yew/tests/hydration.rs

```diff
@@ -913,3 +914,67 @@ async fn hydration_order_issue_nested_suspense() {
         r#"<div>0</div><div>1</div><div>2</div><div>3</div><div>4</div><div>5</div><div>6</div><div>7</div><div>8</div><div>9</div>"#
     );
 }
+
+#[wasm_bindgen_test]
+async fn hydration_props_blocked_until_hydrated() {
+    #[function_component(App)]
+    pub fn app() -> Html {
+        let range = use_state(|| 0u32..2);
+        {
+            let range = range.clone();
+            use_effect_with_deps(
+                move |_| {
+                    range.set(0..3);
+                    || ()
+                },
+                (),
+            );
+        }
+
+        html! {
+            <Suspense>
+                <ToSuspend range={(*range).clone()}/>
+            </Suspense>
+        }
+    }
+
+    #[derive(Properties, PartialEq)]
+    struct ToSuspendProps {
+        range: Range<u32>,
+    }
+
+    #[function_component(ToSuspend)]
+    fn to_suspend(ToSuspendProps { range }: &ToSuspendProps) -> HtmlResult {
+        use_suspend(100)?;
```

Comment:
Nit: I'd change the argument of `use_suspend` to `Duration` so it's immediately clear this is millis.

The exact durations will get replaced either way soon.